ibm/service/vpc/data_source_ibm_is_backup_policy_jobs.go

```diff
@@ -187,6 +188,78 @@ func DataSourceIBMIsBackupPolicyJobs() *schema.Resource {
 										Computed:    true,
 										Description: "The unique user-defined name for this volume.",
 									},
+									"remote": &schema.Schema{
```

Comment:
same as per `ibm/service/vpc/data_source_ibm_is_backup_policy_job.go`

Reply:
Not part of this SRB, still added to PR as it is part of GA feature